BUILDING.md

```diff
@@ -40,21 +42,21 @@ Exokit is a node module. The core is written in Javascript, with native bindings
 
 > *NOTE: Due to combined usage of Ubuntu and Windows tools, you must check out Exokit to somewhere reachable from both; the easiest way is to do your `git clone` inside Ubuntu bash, since for example Ubuntu can reach `C:\exokit` as `/mnt/c/exokit`. (included in instructions below)*
 
-### Procedure (signed)
+### Procedure (unsigned)
```

Comment:
Thanks, just one thing: `signed` is the common case so we should probably keep that as the first one listed.

Reply:
Fixed accordingly